docs/_docs/quickstart.md

```diff
@@ -7,8 +7,11 @@ permalink: /docs/quickstart/
 If you already have a full [Ruby](https://www.ruby-lang.org/en/downloads/) development environment with all headers and [RubyGems](https://rubygems.org/pages/download) installed (see Jekyll's [requirements](/docs/installation/#requirements)), you can create a new Jekyll site by doing the following:
 
 ```sh
-# Install Jekyll and Bundler gems through RubyGems
-gem install jekyll bundler
+# Install Bundler through RubyGems
+gem install bundler
+
+# Add Jekyll to your Gemfile and install its dependencies
+bundle add jekyll && bundle install
```

Comment:
One would need to run `bundle init` or be certain they have a valid `Gemfile` before they can run `bundle add`

Reply:
I’m not sure where this Gemfile would exist. Some directory that is a parent of _all_ of a users Jekyll sites?

Reply:
This is why *the docs currently recommend* the `jekyll new` approach after `gem install jekyll`. Its a lot simpler..

Reply:
I considered whether or not to expicitly mention `bundle init` here, as it is already described in some detail on the tutorial that we will be linking. Is that just repeating ourselves for no reason?

<img width="668" alt="screen shot 2018-04-16 at 3 40 17 pm" src="https://user-images.githubusercontent.com/1534882/38831157-838f676e-418c-11e8-8696-8e17a0d946d2.png">

Reply:
`bundle add jekyll` is detailed in that tutorial as well.. So, your concern is actually moot since `bundle init` is a necessary step for `bundle add jekyll` to run successfully..